warehouse/query-core/src/main/java/datawave/query/tracking/ActiveQueryLog.java

```diff
@@ -68,20 +78,49 @@ private ActiveQueryLog(AccumuloConfiguration conf) {
         }
     }
     
+    synchronized private void touch() {
```

Comment:
since we are simply changing a long here, I am not sure we need to synchronize.  If two threads are setting this at the same time, then it really does not matter that they execute simultaneously.  The setting of the long value is atomic.